localnet/init/config/Settings.toml

```diff
@@ -57,8 +57,7 @@ http_endpoint = "http://localhost:8899"
 ws_endpoint = "ws://localhost:8899"
```

Comment:
shall this like be removed too?

Reply:
Indeed... snuck through the git merge, nice catch